apps/api/src/domain/models/photograph/commands/create-photograph/create-photograph.command-handler.ts

```diff
@@ -19,23 +19,21 @@ import { PhotographCreated } from './photograph-created.event';
 export class CreatePhotographCommandHandler extends BaseCreateCommandHandler<Photograph> {
     protected createNewInstance({
         aggregateCompositeIdentifier: { id },
-        title: titleText,
+        title,
         languageCodeForTitle,
         mediaItemId,
-        photographer,
         heightPx,
         widthPx,
     }: CreatePhotograph): ResultOrError<Photograph> {
         const newInstance = new Photograph({
             type: AggregateType.photograph,
             id,
-            title: buildMultilingualTextWithSingleItem(titleText, languageCodeForTitle),
+            title: buildMultilingualTextWithSingleItem(title, languageCodeForTitle),
```

Comment:
I think I'm ok with this. We were trying to emphasize that the "title"  property is just plain text, not multilingual text as is the `title` on the domain model.